tools/webpack-jumpstart

```diff
@@ -41,7 +53,7 @@ for try in $(seq 50 -1 0); do
     if fetch_to_cache tag "${tag}"; then
         break
     fi
-    if [ "${1-}" != '--wait' -o "$try" = '0' ]; then
+    if [ -z "${wait-}" -o "$try" = '0' ]; then
```

Comment:
I'd prefer you wouldn't use the `-` fallback for undefined variables here, to better spot typos. You explicitly initialize the known options to empty strings (also in the subsequent commit).

Reply:
Good catch.  I didn't do the initialization before, but then I thought about how they might sneak in via the environment, so I added it.  Then I forgot to remove this.